webrtc/RTCPeerConnection-setRemoteDescription.html

```diff
@@ -47,6 +47,43 @@
       assert_unreached('Error ' + e.name + ': ' + e.message);
     }));
   }, 'Triggers ontrack when called with a remote description and the MSID is parsed.');
+
+  promise_test(() => {
+    const pc = new RTCPeerConnection();
+    return pc.setRemoteDescription(
+      {'type': 'offer', 'sdp': 'bogus'})
+    .then(() => {
```

Comment:
This would be a good fit for `promise_rejects`, see https://github.com/w3c/web-platform-tests/blob/master/fetch/api/basic/error-after-response.html for an example.